website/src/website/data/playground-samples/interacting-with-the-editor/adding-a-command-to-an-editor-instance/sample.js

```diff
@@ -32,11 +32,12 @@ editor.addCommand(
 	},
 	"myCondition1 && myCondition2"
 );
-
+// @ts-ignore
```

Comment:
myConditions is typed as `ContextKey<false>` so `true` cannot be assigned to `false`.
If this is typescript, there is another WA. But I can't find it in JS mode, so I added `@ts-ignore`

Reply:
Use put `/** @type ContextKey<boolean> */` before `myCondition1/2`